docs/what-if/deploy-to-uat.md

```diff
@@ -0,0 +1,38 @@
+# What If I want to deploy to UAT?
+
+## How to Deploy specific branch to Test Environment 
+
+1. Get Rundek access first
+
+2. Then create public key in `~/.ssh`
+
+3. Now go to [https://idm.onehippo.com/pubkeys](https://idm.onehippo.com/pubkeys) & add public key
+
+4. Create a file `~/nhsd.rundec.env`
+
+5. Then Add following…
+
+		export RD_URL=https://deploys.onehippo.com/api/21
+		export RD_USER=
+		export RD_PASSWORD=
+		export RD_AUTH_PROMPT=false
+		export RD_COLOR=0
+
+
+6. Checkout the branch you want to deploy 
+
+		cd ci-cd
+
+		make clean build ondemand.upload ondemand.deploy ENV=tst RD_CONF=~/nhsd.rundec.env
+
+7. Now see the status here: 
+
+		https://deploys.onehippo.com/project/nhs/activity
+      		/hippo:moduleconfig:
```

Comment:
from this line down, looks like copy and paste from another document :)

Reply:
I'm sorry @wojtek, I'm not sure if I understand what you're referring to. This is a 'how-to' document with the information we have gathered around deploying to UAT, so that we have a common understanding of the process.

What parts of the PR are you requesting to change please?

Reply:
He means from here down in the document is talking about AutoExport, it looks like a copy paste error!

Reply:
Also, should this read release to test rather than UAT? UAT release is automatic

Reply:
I updated the md file to reflect your requested changes. 'UAT' has been replaced with 'Test' in the file name and in context, and we updated the documentation, so hopefully it makes more sense now. Thanks for the feedback boys!